crates/evm/fuzz/src/strategies/mutators.rs

```diff
@@ -504,30 +502,32 @@ mod tests {
     #[test]
     fn test_mutate_address() {
         let mut runner = TestRunner::new(Config::default());
+        let value = Address::random();
+        let validate_mutation = |mutated: Option<Address>| {
```

Comment:
i think you want this to be a fn with #[track_caller], also could add value itself to the panic message like `{value} -> {mutated:?}`